internal/repository/root_test.go

```diff
@@ -43,6 +44,116 @@ func TestInitializeRoot(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestAddRootKey(t *testing.T) {
+	r, keyBytes := createTestRepositoryWithRoot(t, "")
+
+	key, err := tuf.LoadKeyFromBytes(keyBytes)
```

Comment:
feel free to set up a helpers_test.go file in this package for the `// embed` directives, similar to the policy package. https://github.com/gittuf/gittuf/blob/main/internal/repository/targets_test.go#L20 has a single directive right now.

Reply:
I have added a helper_test.go file, with all test repo creation functions, and all embedded variables in it